OpenRA.Mods.Common/Graphics/DefaultSpriteSequence.cs

```diff
@@ -126,6 +134,27 @@ public DefaultSpriteSequence(ModData modData, TileSet tileSet, SpriteCache cache
 				sprites = cache[src].Select(
 					s => new Sprite(s.Sheet, s.Bounds, s.Offset + offset, s.Channel, blendMode)).ToArray();
 
+				if (FlipX != null)
+				{
+					spritesFlipX = cache[src].Select(
+						s => new Sprite(s.Sheet, Rectangle.FromLTRB(s.Bounds.Right, s.Bounds.Top, s.Bounds.Left, s.Bounds.Bottom),
+							s.Offset + offset, s.Channel, blendMode)).ToArray();
+				}
+
+				if (FlipY != null)
```

Comment:
no. both will be created. 
Yes memory overhead here. but current sprites array also produce a lot of unused entries in it